config.json

```diff
@@ -1454,6 +1454,16 @@
         "filtering",
         "games"
       ]
+    },
+    {
```

Comment:
@tejasbubane do we still use the script to auto sort this file? If not, this is fine :)

Reply:
I don't think we have anything to sort it ATM - I think this is fine.